docs/development/releasing.rst

```diff
@@ -274,30 +277,43 @@ The procedure for this is straightforward:
 
       $ git branch v<version>.x
 
-#. Render the changelog with towncrier, as above.
-
-#. Tag this commit using the next major version followed by ``.dev``. For example,
-   if you have just branched ``4.0``, create the ``v4.1.dev`` tag on the commit
-   adding the ``4.1`` section to the changelog::
-
-      $ git tag -s "v<next_version>.dev" -m "Back to development: v<next_version>"
-
-#. Also update the "what's new" section of the docs to include a section for the
+#. Update the "what's new" section of the docs to include a section for the
```

Comment:
```suggestion
#. Create a new branch from main at the point you want the feature freeze to
   occur and check it out::

      $ git checkout -b v<version>-changelog
      
#.  Render the changelog in the maintainience branch with towncrier, as above. 
      
#.  Create a PR with that changelog, for others to review. Changes in that branch should *not* be made it the rendered version, however, rather they should go in the fragments and later back-ported to main and the release branches.
      
#.  Switch back to the main branch::

      $ git checkout main

#. Update the "what's new" section of the docs to include a section for the
```
I'm not sure if this is the optimal solution, but *some* sort of review of the changelog should happen between now and the RC.

Reply:
I'll open a separate PR for now with something along these lines but with a modification (just in the interest of not having one giant PR for all the release doc updates)